bin/main.rkt

```diff
@@ -266,6 +266,15 @@
                             keywords-sorted))
        (list (cons ',(string->symbol (verilog-module-out-signal)) (bv->signal ,body)))))
   (eval out-fn ns))
+  
+
+
+(define (get-log-filename verilog-module-filepath)
+  (let* ((basename (regexp-replace #rx".*/" verilog-module-filepath "")) ; Remove directory path
+         (name-no-ext (regexp-replace #rx"\\.sv$" basename ""))          ; Remove .sv extension
```

Comment:
i would avoid using regexes here -- there should be Racket tools for handling filepaths without needing to parse them with regexes!

Furthermore, avoid assuming that the file will end with .sv! it could end with .v, or really anything else. I would just take the whole filename e.g. `example.sv` and append both a timestamp and a log to it e.g. `example.sv.<timestamp>.log`.